src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -399,10 +399,11 @@ def add_photon_series(
     metadata_copy = dict_deep_update(
         get_nwb_imaging_metadata(imaging, photon_series_type=photon_series_type), metadata_copy, append_list=False
     )
-    if photon_series_type == "TwoPhotonSeries":
-        assert (
-            "OnePhotonSeries" not in metadata_copy["Ophys"]
-        ), "Received metadata for 'OnePhotonSeries' but `photon_series_type` was not explicitly specified."
+
+    if photon_series_type == "TwoPhotonSeries" and "OnePhotonSeries" in metadata_copy["Ophys"]:
```

Comment:
What about the other way? `photon_series_type == "OnePhotonSeries"` but `"TwoPhotonSeries" in metadata_copy["Ophys"]`?

Reply:
I can add a similar condition for this case. I think the original purpose was to signal that the default value of `photon_series_type`  ("TwoPhotonSeries") was used, which is probably why there wasn't a check the other way around.